Readme.md

```diff
@@ -40,7 +40,7 @@ app.listen(3000);
 ```
 var options = { 
   useErrorHandler: false, 
-  continueMiddleware: false,
+  continueMiddleware: false
```

Comment:
What error did you fix? 

[Trailing commas are valid in JavaScript.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas).

Including them can help prevent some problems. If every object pair ends in a comma, then you don't have to worry about adding a new row and suddenly having an error that the second-row-from-the-last is now missing a required comma.